torch/csrc/autograd/python_variable.cpp

```diff
@@ -1162,6 +1164,48 @@ int THPVariable_set_backwards_hooks(
   END_HANDLE_TH_ERRORS_RET(-1)
 }
 
+PyObject* THPVariable_get_post_accumulate_grad_hooks(
+    THPVariable* self,
+    void* unused) {
+  HANDLE_TH_ERRORS
+  if (check_has_torch_function((PyObject*)self)) {
+    return handle_torch_function_getter(self, "_post_accumulate_grad_hooks");
+  }
+  if (self->post_accumulate_grad_hooks) {
+    Py_INCREF(self->post_accumulate_grad_hooks);
+    return self->post_accumulate_grad_hooks;
+  }
+  Py_RETURN_NONE;
+  END_HANDLE_TH_ERRORS
+}
+
+int THPVariable_set_post_accumulate_grad_hooks(
+    THPVariable* self,
+    PyObject* obj,
+    void* unused) {
+  HANDLE_TH_ERRORS
+  if (check_has_torch_function((PyObject*)self)) {
+    return handle_torch_function_setter(
+        self, "_post_accumulate_grad_hooks", obj);
+  }
+  THPUtils_assertRet(
+      -1, obj, "Deletion of _post_accumulate_grad_hooks not allowed!");
+  if (obj == Py_None) {
+    obj = nullptr;
```

Comment:
Are we actually ok with deleting that field? The check above seems to be against that?

Reply:
Preexisting? To mirror what register_hook does today, the register_hook function assumes that only valid states are None and dict. If None, we set a new OrderedDict.

Reply:
But we don't actually set it to None, we delete it here! haha

Reply:
hmmm, apparently setting obj=nullptr doesn't actually delete the entry?
```
a = torch.tensor(1., requires_grad=True)
a._backward_hooks = None
print(a._backward_hooks)  # None
```

Reply:
That's because our accessor above converts the nullptr back to None :D 

Anyway. We can keep the same behavior for consistency.

Reply:
yay